services/core-web/src/components/noticeOfWork/applications/FinalPermitDocuments.js

```diff
@@ -111,6 +117,7 @@ export class FinalPermitDocuments extends Component {
           editPreambleFileMetadata={this.props.editPreambleFileMetadata}
           isFinalPackageTable
           isAdminView
+          isSortingAllowed={!this.props.adminView && draftInProgress}
         />
```

Comment:
Is sorting is enabled on the manage documents tab, draft tab, but not the admin tab?

Reply:
Enabled just on the Draft tab. The adminView prop is set to true for the Manage Docs and Admin Tab.